components/intro-video/intro-video.njk

```diff
@@ -1,9 +1,10 @@
 <div class="intro-video" role="presentation">
-  <video class="intro-video__video" src="static/video/test_bg.mp4" poster="static/images/eye-placeholder.jpg" muted autoplay loop>
-    <source src="static/video/test_bg.webm" type="video/webm">
-    <source src="static/video/test_bg.mp4" type="video/mp4">
-    <source src="static/video/test_bg.ogv" type="video/ogg">
+  <video class="intro-video__video" poster="static/images/eye-placeholder.jpg" muted autoplay loop>
+    <source data-src="static/video/test_bg.webm" type="video/webm">
+    <source data-src="static/video/test_bg.mp4" type="video/mp4">
+    <source data-src="static/video/test_bg.ogv" type="video/ogg">
```

Comment:
Does it work on iOS Safari? I'm asking because on Explorify we only have mp4 and it's not showing up on iOS.

Reply:
yeah, iOS Safari doesn't allow autoplaying videos – playback has to be triggered by a user action. There are ways around it, like you trigger it via the `touchstart` event :) but i'm not too concerned with that atm, the background video isn't essential.